simplq/src/components/common/slider/slider.stories.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```

Comment:
It should be Slider.stories, right? Capital S like in index.

Reply:
Done